regtests/ww3_ts1/input_nl5_gaussian/ww3_grid.inp

```diff
@@ -0,0 +1,76 @@
+$ -------------------------------------------------------------------- $
```

Comment:
These tests are not added to matrix.base.  I noticed you said these ran a very long time, but if there's no testing at all there is no way to know if we break this functionality moving forward.  Is there a way to do an extremely small test in time so we at least say go through 1 or 2 time steps as a purely technical test? Maybe adding OMP could help?

Reply:
Hi Jessica,

thank you very much for all the comments above. I have revised the code/regtests as suggested when necessary. 

Regarding the matrix.base, I added another simple test folder `input_nl5_matrix` under `ww3_ts1` (it only takes less 1s for this very simple setup) and have updated the `matrix.base` accordingly.

Cheers,
Qingxiang

Reply:
Thanks for adding this to the matrix!